plasma_framework/contracts/src/exits/payment/controllers/PaymentProcessStandardExit.sol

```diff
@@ -29,6 +29,11 @@ library PaymentProcessStandardExit {
         uint256 amount
     );
 
+    event BountyRewardFailed(
```

Comment:
I think this should be called `BountyReturnFailed` for consistency with the other events